packages/core/types/index.js

```diff
@@ -350,8 +348,9 @@ export type BundleGroup = {
 
 export interface BundleGraph {
   findBundlesWithAsset(asset: Asset): Array<Bundle>;
-  getBundleGroups(bundle: Bundle): Array<BundleGroup>;
-  getBundles(bundleGroup: BundleGroup): Array<Bundle>;
+  getBundleGroupsContainingBundle(bundle: Bundle): Array<BundleGroup>;
```

Comment:
There are now two methods for getting bundle groups for a bundle: one retrieving the bundle groups that contain the bundle, and the other for retrieving groups within the bundle (that the bundle depends on).

Too verbose? Thoughts on better names?